functions/AutoRemediateCT-001.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+const AWS = require("aws-sdk");
+
+/**
+* Lambda function to automatically remediate CloudTrail not Enabled
+*/
+module.exports.handler = (event, context, callback) => {
+
+	console.log('CloudTrail not Global - Received event:', JSON.stringify(event, null, 2));
+
+	if (!event || !event.region || event.ruleId !== "CT-001") {
+		return handleError("Invalid event");
+	}
+
+	var cloudtrail = new AWS.CloudTrail({apiVersion: '2013-11-01'});
+	cloudtrail.updateTrail({
+
+		Name: 'GlobalTrail',
+		S3BucketName: 'cc-remediate-cloudtrail',
```

Comment:
Please externalise configurations (such as S3 Buckets) in config.js file. 
https://github.com/cloudconformity/auto-remediate/blob/master/functions/config.js (Line: 10)
https://github.com/cloudconformity/auto-remediate/blob/master/functions/AutoRemediateConfig-001.js (Line: 40)